config/diaspora.yml.example

```diff
@@ -410,6 +410,23 @@ configuration: ## Section
     ## please consider resharing diaspora* HQ's posts for your pod's users!
     #autofollow_on_join_user: 'diasporahq@joindiaspora.com'
 
+    ## Welcome Message on registration (default=false)
+    ## Send a message to new users after registration
+    ## to tell them about your pod and how things
+    ## are handled on it.
+    #welcome_message=false
+
+    ## Welcome Message text (default='Hello [USERNAME], welcome to diaspora.')
+    ## The content of your welcome message.
+    ## The placeholder '[USERNAME]' will be replaced by the username
+    ## of the new user.
+    #welcome_message_text='Hello [USERNAME], welcome to diaspora.'
+
+    ## Welcome Message subject (default='Welcome Message')
+    ## The subject of the conversation that is started
+    ## by your welcome message.
+    #welcome_message_subject='Welcome Message'
+
```

Comment:
I think this would make a nice subsection.

```yaml
welcome_message:
  enabled: false
  subject: "..."
  text: "..."
```

Also note that `foo=bar` is not valid YAML afaik.